students/jemielity_kamil/lesson_02_flow_control/chocolate_bar.py

```diff
@@ -0,0 +1,19 @@
+
+n = int(input("Length of chocolate: "))
+m = int(input("Width of chocolate "))
+k = int(input("Number of squares: "))
+
+if k > n*m:
```

Comment:
It's not stated in PEP8 but it is good practice to use space between math operators 'if k > n * m:'